src/components/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,95 @@
+import * as Polymorphic from '@radix-ui/react-polymorphic'
+import React, { forwardRef } from 'react'
+import { CSS, StitchesVariants, styled } from 'stitches.config'
+import { Box } from '../Box'
+import { Button } from '../Button'
+import { ChevronLeft, ChevronRight } from '../Icons'
+
+const commonButton: CSS = {
+  width: '1em',
+}
+
+export interface PaginationProps {
+  /*
+    The total number of pages
+  */
+  count: number
+  /*
+    Callback fired when the page is changed
+  */
+  onChange?: (event: React.MouseEvent, page: number) => void
```

Comment:
Let's match the API of the other so either `onPageChange`, or use `value` and `onValueChange`.
Also, I don't think we really need the mouse event, have you ever had to do anything with that?

Reply:
Very rarely - I'll remove it.